docs/kafka-connectivity-faq.md

```diff
@@ -0,0 +1,79 @@
+# Kafka External Stream Troubleshooting Guide
+
+## FAQ: Common Kafka External Stream Issues
```

Comment:
use `{#name}` syntax to create more readable section name. Same suggestion for rest of the h2/h3 headers